content/general/concepts/unit-testing/unit-testing.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Unit Testing'
+Description: 'Unit testing involves testing parts of code to verify its behavior.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-javascript-unit-testing'
+---
+
+Unit testing is an essential part of the coding process. It verifies that all parts of the code work correctly.
+
+A part or unit of code can be anything from a class function to a small snippet of code that performs a specific task. Testing individual components makes it easier to find exactly where an error lies.
+
+There are usually two types of unit testing:
```

Comment:
```suggestion
Unit testing is usually categorized into two types:
```

Reply:
@Dusch4593 I kind of liked mentioning the types in the introduction. It made the intro flow nicely with the rest of the entry. But if you think this is a better format, let's change it then.

Reply:
@KTom101 I definitely understand this preference. I'm going to follow @Dusch4593's suggestion however because it provides more structure and clarity. Ty for understanding! Also because some time has passed, I'm also going to incorporate the other heading changes that relate to this as well 🚀